governance-exercises/JanSky Polling Exercise 2 - Forum Post 2.md

```diff
@@ -0,0 +1,16 @@
+# Forum Response 2
+
+Author: JanSky  
+Date: Friday 28th July 2023
+
+---
+
+@Diane I understand your concerns and questions regarding the situation with Clive. It's important to approach matters like this with respect and a commitment to the principles of the MakerDAO community.
+
+Regarding your question about the bounty process, it's important to note that any decision on potential actions, including bounties, will be contingent on the outcome of the poll. I will initiate the poll on Monday to address the issue and will respect the collective decision of MKR holders.
```

Comment:
I would strongly recommend separating the bounty payment from the poll outcome. The scope and atlas make payment of the bounty optional (lots of 'can' in those clauses). You can use this freedom to more intentionally incentivize future doxxing events going in a more ethical direction. 

In this situation, I would likely reward Diane a smaller percentage of the bounty, perhaps 20%, and state that the full bounty will be paid out in case of _responsible_ disclosure.